src/sentry/static/sentry/app/views/eventsV2/eventDetails/content.tsx

```diff
@@ -146,8 +162,8 @@ class EventDetailsContent extends AsyncComponent<Props, State & AsyncComponent['
               projectId={this.projectId}
               eventView={eventView}
             />
-          </Main>
-          <Side>
+          </div>
+          <div style={{gridColumn: '2/3', display: isSidebarVisible ? '' : 'none'}}>
```

Comment:
Could we use a styled component for this. Inline styles get unwieldy quick.

Reply:
Yeah - I tried to do it in a styled component but it wasn't working. Will try this in the new branch.